graphql-parser/src/lexer.rs

```diff
@@ -342,7 +352,8 @@ mod tests {
   }
 
   fn test_str(input: &str, expeced: &str) {
-    test_with_lex(input, Token::String(expeced));
+    let len = expeced.len();
+    test_with_lex(input, StringValue(expeced, 1, len + 1))
```

Comment:
Ok, I have `len + 1` here to make it work, but WHY? This is might be related to the eof issue.